jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/IngressEnricherTest.java

```diff
@@ -210,7 +213,7 @@ void getIngressTlsXMLConfig_withNonNullResourceConfig() {
         List<IngressTlsConfig> ingressTlsConfigs = IngressEnricher.getIngressTlsXMLConfig(resourceConfig);
 
         // Then
-        assertThat(ingressTlsConfigs).asList().hasSize(1);
+        assertThat(ingressTlsConfigs).asInstanceOf(InstanceOfAssertFactories.list(IngressTlsConfig.class)).hasSize(1);
```

Comment:
I think `asList()` call is redundant here. Could you please check if it works without `asList()` ?

Reply:
agree, tested working fine. I will modify this.